src/main/java/wromaciej/hvac_sim/thermo/matter/fluids/service/FluidData.java

```diff
@@ -65,6 +65,7 @@ public Parameter findRefrigerantParameter(ParameterType wantedParameterType, Flu
         return findParameter(wantedParameterType, fluidName, knownParameter1, knownParameter2);
     }
 
-    public FluidData() {
+    public FluidData(boolean shouldLoadLibrary) {
+        if (shouldLoadLibrary) loadLibrary();
```

Comment:
actually you could use something like: https://www.chilkatsoft.com/java-loadLibrary-Linux.asp, so static block. And that is the only static element of the language I you should so far use ;p